src/app/modules/shared-components/components/add-content/add-content.component.ts

```diff
@@ -12,33 +12,58 @@ export class AddContentComponent implements OnInit, OnDestroy {
   @Input() allowSkills = false;
   @Output() contentAdded = new EventEmitter<{ title: string, type: ItemType }>();
 
-  newContentForm: FormGroup = this.formBuilder.group({ title: '' });
+  newContentForm: FormGroup = this.formBuilder.group({ newTitle: '', existingTitle: '' });
+
+  expanded = false;
+  focused: 'existingTitle' | 'newTitle' | null = null;
```

Comment:
You never put this variable back to null. In practice it means the input stays highlighted, even after a clear and doing something else.
BTW, why null and not undefined? Undefined is better in general in TS.

Reply:
Fixed